catalog/core/catalog-core-validationparser/src/main/java/ddf/catalog/validation/impl/ValidationParser.java

```diff
@@ -158,6 +173,10 @@ private void apply(File file) throws Exception {
         handleSection(changeset, "Validators", outer.validators, this::parseValidators);
         handleSection(changeset, "Defaults", outer.defaults, this::parseDefaults);
         handleSection(changeset, "Injections", outer.inject, this::parseInjections);
+        handleSection(changeset,
+                METACARD_VALIDATORS_PROPERTY,
+                outer.metacardValidators,
+                this::registerMetacardValidators);
```

Comment:
A little concerned with the consistency of how things are processed... but not that this is at fault.  Just curious that parseValidators is called seemingly twice, once via handleSection and once above... and the metacardValidators is broken into two parts, one to parse above, and then this to actually create the callable and have handleSection invoke it.